clients/replication/replication.go

```diff
@@ -17,12 +18,25 @@ type Client struct {
 
 type CreateParams struct {
 	clients.OrgParams
-	Name           string
-	Description    string
-	RemoteID       string
-	LocalBucketID  string
-	RemoteBucketID string
-	MaxQueueSize   int64
+	Name                 string
+	Description          string
+	RemoteID             string
+	LocalBucketID        string
+	RemoteBucketID       string
+	MaxQueueSize         int64
+	DropNonRetryableData string
```

Comment:
I would push back on this, I think it's a more idiomatic CLI experience to have both `--flag` and `--no-flag` for bool toggles. Those could be consolidated down to a single `*bool` in this struct without a ton of custom code at the `cmd/` layer.

Reply:
Made updates so that this works with a `--drop-non-retryable-data` and `--no-drop-non-retryable-data` flag now. I didn't end up using a `*bool` here since I would need to store the value for both flags and compute the correct pointer value for those somewhere, and it seemed a little easier to do that in the client code.